Controllers/Frontend/StripePayment.php

```diff
@@ -298,6 +379,74 @@ protected function saveOrderWithCharge(Stripe\Charge $charge)
         return $order;
     }
 
+    /**
+     * Saves the order in the database adding the ID of the given $source (as 'paymentUniqueId' aka 'temporaryID').
+     *
+     * @param Stripe\Source $source
+     * @return Order
+     */
+    protected function saveOrderWithSource(Stripe\Source $source)
+    {
+        // Save the order with a temporary transactionId
+        $orderNumber = $this->saveOrder(
+            'src_pending_'. mb_substr($source->id, 4), // transactionId
```

Comment:
I remember discussing this. But looking at it now I realized that we don't have to store the source ID in `transactionId`, since we always also store it in `paymentUniqueId` (aka `Order::temporaryId`). So we can just change this as follows and retrieve the source ID from `temporaryId` if necessary:
```suggestion
            'stripe_source_pending', // transactionId
```

Reply:
As discussed we need to use a unique value for the transactionId which is why we choose to use a prefix and the `sourceId` so we can later identify which orders have a pending source. 

We did change it a bit and the format now is `pending_src_7123123120123`.